content/blog/2021-04-15-april-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,182 @@
+﻿---
+title: April '21 Heartbeat
+date: 2021-04-15
+description: |
+  Monthly updates are here! Loads from the Community, use cases, videos, 5 new hires and more!
+descriptionLong: |
+  Monthly updates are here! Read all about what our community is doing
+  with DVC, our CEO's interviews and talks, new team members and more!
+picture: 2021-04-15/apr21cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/april-21-heartbeat/718?u=jendefig
+tags:
+  - Heartbeat
+  - DVC
+  - MLOps
+---
+
+## From the Community
+
+We're starting with the community this month because it has been overflowing
+with great content from our users. It's like we're on a sugar high!
+
+![Sugar High](https://media.giphy.com/media/oiGCnybFPh6Q8/giphy.gif)
+
+### Goku Mohandas Tutorial
```

Comment:
I think he calls them lessons (it's more of a course) - we can use the same terminology. May be mentioned that it is part of the course